app.R

```diff
@@ -0,0 +1,1267 @@
+          p( # shinyBS
+            class = "hangingindent",
+            "Bailey, E. (2015). shinyBS: Twitter Bootstrap Components for Shiny. R package version 0.61. Available from https://CRAN.R-project.org/package=shinyBS"
+          ),
+          p( # boastUtils
+            class = "hangingindent",
+            "Carey, R. and Hatfield, N. (2020). boastUtils: BOAST Utilities. R package version 0.1.6.3. Available from https://github.com/EducationShinyAppTeam/boastUtils"
+          ),
+          p( # shiny
+            class = "hangingindent",
+            "Chang, W., Cheng, J., Allaire, J., Xie, Y., and McPherson, J. (2020). shiny: Web Application Framework for R. R package version 1.5.0. Available from https://CRAN.R-project.org/package=shiny"
+          ),
+          p( # shinydashboard
+            class = "hangingindent",
+            "Chang, W. and Borges Ribeiro, B. (2018). shinydashboard: Create Dashboards with 'Shiny'. R package version 0.7.1. Available from https://CRAN.R-project.org/package=shinydashboard"
+          ),
+          p( 
+            class = "hangingindent",
+            "Chi-Square Goodness of Fit.” Statistics Online Support, sites.utexas.edu/sos/guided/inferential/categorical/univariate/chi2/"
+          ),
+          p( # shinyjs
+            class = "hangingindent",
+            "Dean Attali (2020). shinyjs: Easily Improve the User Experience of Your Shiny Apps in Seconds. R package version 2.0.0. https://CRAN.R-project.org/package=shinyjs"
+          ),
+          p( # shinyWidgets
+            class = "hangingindent",
+            "Perrier, V., Meyer, F., and Granjon, D. (2020). shinyWidgets: Custom Inputs Widgets for Shiny. R package version 0.5.3. Available from https://CRAN.R-project.org/package=shinyWidgets"
+            ),
+          br(),
+          br(),
+          br(),
+          boastUtils::copyrightInfo()
+        )
+      )
+    )
+  )
+)
+
+# Define server logic ----
+server <- function(input, output, session) {
+    ## Bottons Output ----
+    ### Info button
+    observeEvent(
+      eventExpr = input$info,
+      handlerExpr = {
+        sendSweetAlert(
+          session = session,
+          type = "info",
+          title = "Information",
+          text = "You will explore Chi-Square Goodness-fit-Test with simulations in this app",
+          btn_colors = "black"
+        )
+      }
+    )
+  
+    ### Button on each page
+   # Onverview
+    observeEvent(
+      eventExpr = input$overviewBotton,
+      handlerExpr = {
+      updateTabItems(
+        session = session,
+        inputId = "pages",
+        selected = "prerequisites")
+    })
+    # Prerequisites
+    observeEvent(
+      eventExpr = input$prerequisitesBotton,
+      handlerExpr = {
+    updateTabItems(
+      session = session,
+      inputId = "pages",
+      selected = "example")
+  })
+    # Explore
+    observeEvent(
+      eventExpr = input$namesPopover,
+      handlerExpr = {
+        sendSweetAlert(
```

Comment:
Since this is a pop up, I'm not sure if it works for accessibility purposes. If wave said it was ok, then no worries! I was just curious since popups do not have alt text.